### PR TITLE
DOCS-265 - Fix for F5 BIG-IP

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request fixes steps here: 
https://help.sumologic.com/docs/integrations/security-threat-detection/f5-big-ip-ltm/#collection-setup

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[DOCS-265](https://sumologic.atlassian.net/browse/DOCS-265)

### CODE REVIEW

**Comment by jpipkin1** [comment]
@carlos-castillo-jask, could you (or someone else on your team) check this PR and let me know if the steps look correct?

File: docs/integrations/security-threat-detection/f5-big-ip-ltm.md [hunk 1]
```diff
@@ -87,25 +87,25 @@ Perform the following tasks to configure log collection for the F5 - BIG-IP LTM
...
+Use the following curl command to set logging profile at BIG-IP. Replace `BIG-IP PWD`, `BIG-IP IP`, and `PORT` in the following command:
 
 ```bash
 curl -k --user admin:<BIG-IP PWD> -H "Accept: application/json" -H "Content-Type:application/json" -X POST -d@AS3Listener.json https://<BIG-IP IP>:<PORT>/mgmt/shared/appsvcs/declare | python -m json.tool
 ```
 
 #### Step 3: Sumo Logic Consumer Set up using Telemetry  
 
-Download the Sumo Logic `sumo.json` [here](https://sumologic-app-data.s3.amazonaws.com/F5LTM/sumo.json), and replace `SUMOLOGIC_HOST` with the value created in step 1, and replace `SUMOLOGIC_SECRET_KEY` with your Sumo Logic [access key](/docs/manage/security/access-keys/).
+Download the Sumo Logic `sumo.json` [here](https://sumologic-app-data.s3.amazonaws.com/F5LTM/sumo.json), and replace `SUMOLOGIC_HOST` and `SUMOLOGIC_SECRET_KEY` with values from Sumo logic HTTP URL created in step 1.
```
Comment (kimsauce) [inline docs/integrations/security-threat-detection/f5-big-ip-ltm.md:104]:
```suggestion
Download the Sumo Logic `sumo.json` [here](https://sumologic-app-data.s3.amazonaws.com/F5LTM/sumo.json), and replace `SUMOLOGIC_HOST` and `SUMOLOGIC_SECRET_KEY` with values from Sumo Logic HTTP URL created in step 1. 
```

File: docs/integrations/security-threat-detection/f5-big-ip-ltm.md [hunk 2]
```diff
@@ -87,25 +87,25 @@ Perform the following tasks to configure log collection for the F5 - BIG-IP LTM
 #### Step 1: Create a Sumo Logic Hosted Collector
 
 1. Configure a Hosted Collector in Sumo Logic using [these instructions](/docs/send-data/hosted-collectors/configure-hosted-collector).
-2. Add an [HTTP source](/docs/send-data/hosted-collectors/http-source/logs-metrics), configuring the Source Category with the string f5 in it (e.g., “f5/ltm”). Make a note of the URL for the HTTP source, as you will need it in the following steps. The URL for our example is: `https://collectors.us2.sumologic.com/receiver/v1/http/`.
+2. Add an [HTTP source](/docs/send-data/hosted-collectors/http-source/logs-metrics), configuring the Source Category with the string f5 in it (e.g., “f5/ltm”). Make a note of the URL for the HTTP source, as you will need it in the following steps. The URL for our example is: `https://collectors.us2.sumologic.com/receiver/v1/http/Thisis4fakeendpoint4testing==`.
```
Comment (kimsauce) [inline docs/integrations/security-threat-detection/f5-big-ip-ltm.md:90]: @carlos-castillo-jask @jpipkin1 Would you be open to something cleaner, like `test-endpoint`?

File: docs/integrations/security-threat-detection/f5-big-ip-ltm.md [hunk 3]
```diff
@@ -87,25 +87,25 @@ Perform the following tasks to configure log collection for the F5 - BIG-IP LTM
...
 #### Step 3: Sumo Logic Consumer Set up using Telemetry  
 
-Download the Sumo Logic `sumo.json` [here](https://sumologic-app-data.s3.amazonaws.com/F5LTM/sumo.json), and replace `SUMOLOGIC_HOST` with the value created in step 1, and replace `SUMOLOGIC_SECRET_KEY` with your Sumo Logic [access key](/docs/manage/security/access-keys/).
+Download the Sumo Logic `sumo.json` [here](https://sumologic-app-data.s3.amazonaws.com/F5LTM/sumo.json), and replace `SUMOLOGIC_HOST` and `SUMOLOGIC_SECRET_KEY` with values from Sumo logic HTTP URL created in step 1. 
 
-Example: For the Sumo Logic HTTP URL `https://collectors.us2.sumologic.com/receiver/v1/http/`, the above values are as follows:
+Example: For the Sumo Logic HTTP URL `https://collectors.us2.sumologic.com/receiver/v1/http/Thisis4fakeendpoint4testing==`, the above values are as follows:
 * `SUMOLOGIC_HOST`. `collectors.us2.sumologic.com`
-* `SUMOLOGIC_SECRET_KEY`. `exampleAccessKey`
+* `SUMOLOGIC_SECRET_KEY`. `Thisis4fakeendpoint4testing==`
```
Comment (kimsauce) [inline docs/integrations/security-threat-detection/f5-big-ip-ltm.md:108]: Same as https://github.com/SumoLogic/sumologic-documentation/pull/4188/files#r1632086615

File: docs/integrations/security-threat-detection/f5-big-ip-ltm.md (hunk 2 above)
Comment (jpipkin1) [inline docs/integrations/security-threat-detection/f5-big-ip-ltm.md:90]:
```suggestion
2. Add an [HTTP source](/docs/send-data/hosted-collectors/http-source/logs-metrics), configuring the Source Category with the string f5 in it (e.g., “f5/ltm”). Make a note of the URL for the HTTP source, as you will need it in the following steps. The URL for our example is: `https://collectors.us2.sumologic.com/receiver/v1/http/test-endpoint`.
```

Comment (jpipkin1) [reply docs/integrations/security-threat-detection/f5-big-ip-ltm.md]: Fixed. You are absolutely right, @kimsauce.

File: docs/integrations/security-threat-detection/f5-big-ip-ltm.md (hunk 3 above)
Comment (jpipkin1) [reply docs/integrations/security-threat-detection/f5-big-ip-ltm.md]: Fixed.

**Comment by jpipkin1** [comment]
@kimsauce, could you take another look at this PR and approve if it looks okay?

**Review by kimsauce — APPROVED** [review]

**Pull request merged**